README.md

```diff
@@ -62,22 +58,21 @@ which will build the ``azavea/TNC-LA-Freshwater`` region, assuming the region ha
 
-The executable installer will be in the ``[workspace]\output`` folder after the script runs successfully.
-
-#### Auto-installing a region
-If the build script is running on a machine with IIS, you can have it automatically remove old versions and install the recently built version in either the `production` or `development` environment.  The command is modified to add the additional `--dev` or `--prod` argument:
+A zipfile with the contents of the site will be in the ``[workspace]\output`` folder after the script runs successfully.
 
-`python build.py gulfmex-region --prod`
+#### Installing a region
 
-The following convention applies for auto-installation:
+The output of the build tool is a zipfile that contains the site contents. The contents are ready to be served from a cloud storage provider like AWS S3 or Azure Storage, or from a webserver like IIS. The steps to get the site hosted for each hosting environment will be different, but the basic idea is that the contents of the zipfile should be extracted to a folder, and then that folder should served as the base directory of the site. See the links below for specific instructions for each provider:
 
-_The repo name contains the url and and install path and ends with `-region`.  For instance `gulfmex-region` will be installed to `C:\projects\TNC\gulfmex` at the url `gulfmex`._
+- [AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html)
+- [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website)
+- [IIS](https://docs.microsoft.com/en-us/iis/manage/creating-websites/scenario-build-a-static-website-on-iis)
 
 #### Using the build script to set up a development environment
-The same script leaves all of the intermediate code in the ``[workspace]\build`` directory so that plugin developers need only to create an IIS Application which points to ``[workspace]\build\GeositeFramework\src\GeositeFramework``.  This will be a working version of the region and all of its plugins, ready to be served.
+The same script leaves all of the intermediate code in the ``[workspace]\build`` directory. This will be a working version of the region and all of its plugins, ready to be served.
 
 #### Building multiple regions at once
-The build script can build installers for multiple sites at once by use of a config file and a flag to the script.  If you created a Coastal Resilience file named ``ca.conf`` with the contents being one full Org/Repo per line:
+The build script can build multiple sites at once by use of a config file and a flag to the script.  If you created a Coastal Resilience file named ``ca.conf`` with the contents being one full Org/Repo per line:
```

Comment:
Hm we talked about `conf` only using the default branches (e.g.`master` on the Framework repo) and I've been thinking about that more. Is there a reason we don't allow for the same build.py config changes as building 1 project? I like the idea of allowing the framework branch flag to be fed through. This could be helpful since we will maintain the old IIS setup branch. I think that would make more sense than allowing region branches when there's multiple regions being worked with.

Reply:
We don't allow it only because it hasn't been requested or budgeted for. It wouldn't be a big change to support this, but I think we should defer on it for now until it's requested.